kernel/src/process_standard.rs

```diff
@@ -1350,7 +1409,7 @@ impl<C: 'static + Chip> ProcessStandard<'_, C> {
         // sure we allocate enough memory just for that.
 
         // Make room for grant pointers.
-        let grant_ptr_size = mem::size_of::<*const usize>();
+        let grant_ptr_size = mem::size_of::<(usize, *mut u8)>();
```

Comment:
`*const`

Reply:
This is a pointer to mutable memory, as it points to where we store the upcalls and grant object.

Reply:
Ah, right. Sorry, my pattern-match wasn't selective enough.